lib/refile/s3.rb

```diff
@@ -45,7 +45,6 @@ def initialize(region:, bucket:, max_size: nil, prefix: nil, hasher: Refile::Ran
       @s3 = Aws::S3::Resource.new @s3_options
       credentials = @s3.client.config.credentials
```

Comment:
`credentials = @s3.client.config.credentials.credentials` should work, as I think @evanpon found out.